jni_gen_test/run_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
You could invoke the setup script from the `setUp` in the test, so it is automatically run.
Be sure to print that is running as well for if the setup fails.